src/pages/en/core-concepts/layouts.md

```diff
@@ -62,9 +62,11 @@ Page layouts are especially useful for [Markdown files](/en/guides/markdown-cont
 ```markdown {2}
 ---
 layout: ../../layouts/BlogPostLayout.astro
-title: Blog Post
-description: My first blog post!
----
+title: Astro in brief
+author: Himanshu
+description: Find out what makes Astro awesome!
+--- 
+<!--- Text below will be injected in <slot/>--->
```

Comment:
```suggestion
```
I'm going to take this out of the Markdown file here, @himanshu007-creator. I think I'd like to keep the Markdown file itself very clean, and since this is a section on layouts,  we'll annotate in that file.